### DISCUSSION TITLE
Receive bot messages update

### DISCUSSION THREAD

**Discussion by matheusf31** [yagop/node-telegram-bot-api#1089] [Q&A] [answered]
Is there a way to receive in `bot.on("message", msg => {` when a message is sent from the bot? Now I am not receiving then.

One week ago I use the Telegram Webhook and it works fine. But now I not receiving anymore the bot messages in the webhook.

I create a listener with the basic config (as it is in documentation):
![image](https://github.com/yagop/node-telegram-bot-api/assets/45342938/a65f97bd-10d5-49bf-aa7e-9c0e0d621ec8)

**Comment by hyerdev3** [comment]
I'm also having the same problem :((

**Comment by tonicaval** [comment]
same here

**Comment by danielperez9430** [comment, marked as answer]
Is not possible from your bot get messages from others bots. Is a Telegram Bot API limitation server side. But if you need this updates you need a "user bot", use your telegram account or other as bot and also need to use other library (Example: tdlib, madelineproto, pyrogram, ... or others that support MTProto API - Client API)

> Why doesn't my bot see messages from other bots?
> Bots talking to each other could potentially get stuck in unwelcome loops. To avoid this, we decided that bots will not be able to see messages from other bots regardless of mode.
Reference: https://core.telegram.org/bots/faq#why-doesn-39t-my-bot-see-messages-from-other-bots

**Reply by matheusf31** [reply]
> Is not possible from your bot get messages from others bots.

I want to receive the updates of messages sent by my bot with /sendMessage for example. Is not possible too? You say others bots, but I want updates from messages of my bot.

**Reply by danielperez9430** [reply]
> > Is not possible from your bot get messages from others bots.
> 
> 
> 
> I want to receive the updates of messages sent by my bot with /sendMessage for example. Is not possible too? You say others bots, but I want updates from messages of my bot.

You can't get messages from any bot. The same to your bot.

But went you send a messsage with (sendMessager or other methods) in the .then of the method you can get the answers (msg object, with the id and content that you sent). So this is diferent from the events that Telegram send you (not include messages from any bot).
